clarity/src/vm/functions/conversions.rs

```diff
@@ -260,7 +266,7 @@ pub fn from_consensus_buff(
     // Perform the deserialization and check that it deserialized to the expected
     // type. A type mismatch at this point is an error that should be surfaced in
     // Clarity (as a none return).
-    let result = match Value::try_deserialize_bytes_exact(&input_bytes, &type_arg) {
+    let result = match Value::try_deserialize_bytes_exact(&input_bytes, &type_arg, false) {
```

Comment:
Might be beneficial to add an explanatory comment everywhere you hard-code `sanitize=false`.

Reply:
Yep, good idea -- this site should be epoch-switched anyways, so I updated it in 08fd9f5